pytorch_blade/src/compiler/tensorrt/bridge/tensorrt_onnx_parser.cpp

```diff
@@ -132,7 +132,15 @@ TrtUniquePtr<nvinfer1::ICudaEngine> TensorrtOnnxParser::BuildEngine(
     }
     auto trt_engine = TrtUniquePtr<nvinfer1::ICudaEngine>(
         context.builder->buildEngineWithConfig(*context.network, *config));
-    if (trt_engine == nullptr) {
+
+    // We had meet segfault when retrive error message after build engine
+    // failed. To make our process more robust we would like to enable log
+    // message only when TORCH_BLADE_DEBUG_LOG==on
+    const char* debug_log_flag = std::getenv("TORCH_BLADE_DEBUG_LOG");
+    bool enable_debug_log = debug_log_flag != nullptr &&
+        (std::strcmp(debug_log_flag, "on") == 0 ||
+         std::strcmp(debug_log_flag, "true") == 0);
+    if (trt_engine == nullptr && enable_debug_log) {
```

Comment:
nit: if it's tricky to use tensorflow::ReadBoolFromEnvVar()
We'd better make it into some util files.

Reply:
Done.